### DISCUSSION TITLE
Speed of fetching articles

### DISCUSSION THREAD

**Discussion by ghost** [jocmp/capyreader#278] [Q&A] [answered]
It takes some time to fetch articles from any rss feed. Is this due to the hosting service or the application's delay time?

**Comment by jocmp** [comment, marked as answer]
Ah, this is a bug for local feeds. Each feed is refreshed sequentially when they should all update in parallel. I merged a fix that I'll send out shortly to a dev release https://github.com/jocmp/capyreader/pull/279

**Comment by jocmp** [comment]
Closing as resolved. Feel free to reopen if you see other speed issues.
